src/components/main/welcome/Welcome.module.css

```diff
@@ -0,0 +1,53 @@
+.container {
+  height: 100%;
+  width: 100%;
+  max-height: 58rem;
+  min-height: 46rem;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 5rem;
+}
+
+.container > div {
+  align-items: center;
+}
```

Comment:
I think it's better to use a class for the `div` in these cases. You could apply these styles to other `div` you don't want to this way

Reply:
Perfect, updated!